src/transport/SecurePairingSession.h

```diff
@@ -139,6 +136,30 @@ class DLL_EXPORT SecurePairingSession
      */
     CHIP_ERROR HandlePeerMessage(MessageHeader & header, System::PacketBuffer * msg);
 
+    /**
+     * @brief
+     *  Return the associated secure session peer NodeId
+     *
+     * @return Optional<NodeId> The associated peer NodeId
+     */
+    const Optional<NodeId> & GetPeerNodeId() const { return mPeerNodeId; };
```

Comment:
I'm unfamiliar with this syntax, is better than returning nullptr if mPeerNodeId?

Reply:
I think there will be more code changes as I would need to change mPeerNodeId to be a pointer otherwise.

Reply:
Fixed.